frontend/public/components/modals/_modals.scss

```diff
@@ -114,3 +114,9 @@
 .modal-paragraph {
   margin-bottom: var(--pf-v6-c-content--MarginBottom);
 }
+
+.co-datalist {
```

Comment:
I would put this in https://github.com/openshift/console/blob/29a7a6903ecb503c242880ec4373653fe8a8edbd/frontend/public/style/_overrides.scss#L78 since it's overriding a Bootstrap rule.

Reply:
Updated the location of the override and also moved where it is placed closer to where the override happens (on the DataListCheck instead of the whole DataList)